### ISSUE TITLE
Support for the Person API

### ISSUE THREAD

**Issue by YSavir** [stripe-ruby-mock/stripe-ruby-mock#770]
Are there any plans to support the Person API?

There is currently an open PR (#689) for the relevant endpoints. A bit out of date, but if it was caught up to the current master and had any outstanding issues fixed, would it be mergeable?

**Comment by csalvato** [comment]
If it were brought up to date and the test suite passed, it would be mergeable. Is that something you or @eyefodder can take on?

**Comment by YSavir** [comment]
Absolutely. I'll wait a bit to see if @eyefodder wants to wrap up the functionality, but if not, I'll do what's needed to get it in.
